scripts/ci-footprint-and-performance/setup/021-install-kernel-rt.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+sudo subscription-manager repos --enable rhel-9-for-x86_64-rt-rpms
+sudo dnf install kernel-rt realtime-setup realtime-tests -y
+sudo grubby --set-default="$(ls /boot/vmlinuz*rt)"
```

Comment:
Kernel files tend to accumulate. I understand that we're installing `kernel-rt` at this spot, but can we still make the code more robust? 
Basically, we should 
* rpm -q kernel-rt | sort | tail -1
* Use the value in grubby command